src/snowflake/cli/plugins/snowpark/common.py

```diff
@@ -197,3 +205,17 @@ def format_arg(arg: Argument):
         schema=udf_sproc.schema_name,
     )
     return f"{name}({arguments})"
+
+
+def _get_import_from_resource(
+    resource_json: dict, source_name: str = "app.zip"
```

Comment:
Please do not use hardcoded values like this one. It's often a source of errors

Reply:
Changed